The Laundry/translation.json

```diff
@@ -75,9 +82,10 @@
   "heavy-machine" : "Heavy Machine",
   "height" : "Height",
   "hide" : "Hide",
+  "hp" : "HP",
   "idea" : "Idea",
-  "influence" : "Influence",
-  "initial-san" : "Mythos (00%)",
+  "influence" : "Charisma",
```

Comment:
I don't think this is correct. Either the key is wrong or the text is wrong. If this is the auto-generated json then the text in the HTML is incorrect.